libs/templates/src/lib/home/sections/live-market-section/index.tsx

```diff
@@ -1,7 +1,70 @@
 import { LiveMarket } from '@deriv-com/blocks';
 import { LivePriceData } from './data';
+import { ReactNode, useEffect, useState } from 'react';
+import { LiveMarketContent } from '@deriv-com/components';
+import {
+  MarketForexAudusdIcon,
+  MarketForexEurusdIcon,
+  MarketForexGbpjpyIcon,
+  MarketForexGbpusdIcon,
+  MarketForexUsdcadIcon,
+  StandaloneLocationCrosshairsBoldIcon,
+} from '@deriv/quill-icons';
 
 const LiveMarketSection = () => {
+  const [LivePriceData, setLivePriceData] = useState<LiveMarketContent[]>([]);
+
+  const forexIcons: { [key: string]: ReactNode } = {
```

Comment:
@prince-deriv we need to think about a mapping between, backend and quill-icons project. 
created an issue in deriv-com/quill-icons#5